be/src/connector_sink/hive_chunk_sink.h

```diff
@@ -71,8 +71,9 @@ struct HiveChunkSinkContext : public ConnectorChunkSinkContext {
     std::vector<std::unique_ptr<ColumnEvaluator>> data_column_evaluators;
     std::vector<std::string> partition_column_names;
     std::vector<std::unique_ptr<ColumnEvaluator>> partition_column_evaluators;
-    int64_t max_file_size;
+    int64_t max_file_size = 128L * 1024 * 1024;
     std::string format;
+    TCompressionType::type compression_type;
     std::map<std::string, std::string> options;
     PriorityThreadPool* executor;
```

Comment:
this should be set to null too.
all pointer member should be set to nullptr by default.